.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: [ '2.6.x', '2.5.x', '2.4.x', '2.3.x' ]
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Ruby
+      uses: actions/setup-ruby@v1
+      with:
+        version: ${{ matrix.ruby }}
+    - name: Install dependencies
+      run: |
+        script/ci/install_deps
```

Comment:
Shouldn't this be `run: script/ci/install_deps`?

Also, the `script/travis` directory needs to be renamed. 😉

Reply:
You're not meant to be paying attention yet 😉